app/authentication/auth.py

```diff
@@ -124,6 +139,9 @@ def requires_auth():
 
 def _auth_by_api_key(auth_token):
     try:
+        # in the case where the full API key is provided get the last 36 chars
+        if len(auth_token) > 36:
```

Comment:
I think that you don't need the if in Python but feel free to keep it!

Reply:
Hey @AntoineAugusti not sure what you mean by this ? Is there an alternative to an if statement here ?

Reply:
🤦‍♂️ yeah there is no need for it here